style-dictionary/transforms/size/dp-transitive.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (StyleDictionary) => {
```

Comment:
This is a copy of this transitive transform: https://amzn.github.io/style-dictionary/#/transforms?id=sizedp

I couldn't find a way to make this transitive so I copied the source code and added the `transitive: true` line.